### PR TITLE
Add how to bind enum article

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Port existing article to WPF Desktop [preview](https://review.docs.microsoft.com/en-us/dotnet/desktop/wpf/data/how-to-bind-to-an-enumeration?view=netdesktop-5.0&branch=pr-en-us-1056)

Contributes to #1020

@tdykstra

### CODE REVIEW

File: dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - "vb"
+helpviewer_keywords: 
+  - "binding data [WPF], enumeration"
+  - "data binding [WPF], enumeration"
+  - "enumeration [WPF]"
+---
+
+# How to bind to an enumeration (WPF .NET)
+
+This example shows how to bind to an enumeration. Unfortunately there isn't a direct way to use an enumeration as a data binding source. However, the <xref:System.Enum.GetValues(System.Type)?displayProperty=nameWithType> method returns a collection of values. These values can be wrapped in a <xref:System.Windows.Data.ObjectDataProvider> and used as a data source.
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md:18]:
```suggestion
This example shows how to bind to an enumeration. Unfortunately there isn't a direct way to use an enumeration as a data binding source. However, the <xref:System.Enum.GetValues(System.Type)?displayProperty=nameWithType> method returns a collection of values. These values can be wrapped in an <xref:System.Windows.Data.ObjectDataProvider> and used as a data source.
```

File: dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The <xref:System.Windows.Data.ObjectDataProvider> type provides a convenient way to create an object in XAML and use it as a data source.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md:23]: Should there be a Prerequisites section?

**Review by tdykstra — APPROVED** [review]
LGTM

File: dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md (hunk 2 above)
Comment (adegeo) [reply dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md:23]: I don't think there is anything special they need. Did you have something in mind?

Comment (tdykstra) [reply dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md:23]: No, just asking.

**Pull request merged**